[fix] main_xrc.py: put back white-space in binary data

pieleric opened the pull request:
Commit 194eed4c65e (remove trailing whitespace) cleaned up all the
trailling whitespaces. However, main_xrc.py is automatically generated.
So next time it's generated, the white space will be back, so need to
remove them from source files: the XRC files.

More importantly, some of the "trailing white-space" is part of
multi-line byte strings where every character counts. Removing a
space has a big change in behaviour. => The PNGs got corrupted.

pieleric committed:
[fix] main_xrc.py: put back white-space in binary data

Commit 194eed4c65e (remove trailing whitespace) cleaned up all the
trailling whitespaces. However, main_xrc.py is automatically generated.
So next time it's generated, the white space will be back, so need to
remove them from source files: the XRC files.

More importantly, some of the "trailing white-space" is part of
multi-line byte strings where every character counts. Removing a
space has a big change in behaviour. => The PNGs got corrupted.

patrickcleeve2 commented:
@pieleric I think it might be worth ignoring the autogenerated files (or at least trailing-white-space), so that the linter still passes. 
https://flake8.pycqa.org/en/3.0.1/user/ignoring-errors.html#ignoring-entire-files

pieleric committed:
[ci] disable linting from the auto generated files

The files are typically good, but if there is an issue, we cannot do
much, so it's not helpful to complain.
Also, there is a bug in flake8 which flags trailing whitespace in
multiline strings. So that hides this bug too.
